apps/aecore/src/aec_chain_state.erl

```diff
@@ -431,7 +431,7 @@ internal_insert(Node, Block, Origin) ->
             try aec_db:ensure_transaction(Fun)
             catch exit:{aborted, {throw, ?internal_error(What)}} -> internal_error(What)
             end;
-        {ok, Node} -> ok;
+        {ok, Node} -> {error, already_in_db};
```

Comment:
`internal_insert_transaction` below returns `ok` in one branch - should also be `{error, already_in_db}`?

Reply:
Nevermind, I can't read.